app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +393,47 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
+            <div
+              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
+            >
+              <ReportButton />
+            </div>
+            <div
+              className={classes.menuBtn}
+              onClick={menuOpen ? handleMenuClose : handleMenuOpen}
+            >
+              <MenuIcon />
+              <Avatar src={user?.avatarUrl} className={classes.avatar}>
+                {user?.avatarUrl && user.avatarUrl?.length > 0
+                  ? null
+                  : acronym(user?.name)}
+              </Avatar>
+            </div>
+          </div>
+
+          <Menu
+            open={menuOpen}
+            anchorEl={menuRef.current}
+            elevation={0}
+            onClose={handleMenuClose}
+            classes={{
+              paper: classes.menu,
+            }}
+            anchorOrigin={{
+              vertical: "bottom",
+              horizontal: "right",
+            }}
+            getContentAnchorEl={null}
+            PopoverClasses={{
+              root: classes.menuPopover,
+            }}
```

Comment:
Instead of doing this, do`<Menu className={classes....`. Before there was no reason expect to match the rest of the codebase, but there is actually a very good reason now - we will have to do this when we migrate to MUI v5